x-pack/plugins/lens/public/shared_components/visual_options_popover/line_curve_option.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { CurveType } from '@elastic/charts';
+import { EuiFormRow, EuiIconTip, EuiSwitch } from '@elastic/eui';
+
+export interface LineCurveOptionProps {
+  /**
+   * Currently selected value
+   */
+  value: CurveType;
+  /**
+   * Callback on display option change
+   */
+  onChange: (id: CurveType) => void;
+  isCurveTypeEnabled: boolean;
+}
+
+export const LineCurveOption: React.FC<LineCurveOptionProps> = ({
+  onChange,
+  value,
+  isCurveTypeEnabled,
+}) => {
+  return isCurveTypeEnabled ? (
+    <EuiFormRow
+      display="columnCompressedSwitch"
+      label={
+        <>
+          {i18n.translate('xpack.lens.xyChart.curveStyleLabel', {
+            defaultMessage: 'Line style',
```

Comment:
For consistency with how we've previously presented most switches across Lens, can you change the `EuiFormRow` label to something like `Curve lines` (and subsequently hide the label on the `EuiSwitch` component via the `showLabel` prop)?